addons/mail/static/src/models/attachment/attachment.js

```diff
@@ -400,6 +419,13 @@ function factory(dependencies) {
             compute: '_computeExtension',
         }),
         filename: attr(),
+        /**
+         * Determines if the attachement has link preview informations.
+         */
+        hasLinkPreview: attr({
```

Comment:
hasLinkPreview -> isLinkPreview

In current implementation the attachment is the link preview itself.